client/src/components/AdminPanel/ManageMonuments/ManageMonument/ManageMonument.js

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+import About from '../../../Monument/Details/About/About';
+import { Link } from 'react-router-dom';
+import { Alert, Button, Modal } from 'react-bootstrap';
+
+export default class ManageMonument extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            toggleActiveModalOpen: false,
+            deleteModalOpen: false
+        };
+    }
+
+    toggleActive(active) {
+        const { onToggleActive } = this.props;
+        onToggleActive(active);
+        this.setState({toggleActiveModalOpen: false});
+    }
+
+    deleteMonument() {
+        const { onDeleteMonument } = this.props;
+        onDeleteMonument();
+        this.setState({deleteModalOpen: false});
+    }
+
+    render() {
+        const { monument, deleted } = this.props;
+        if (deleted) return (
+            <div className="manage-monument">
+                <div>
+                    You have deleted {monument.title} successfully.
+                </div>
+            </div>
+        );
+        return (<>
+            {this.renderManageMonument()}
+            {this.renderConfirmToggleActiveModal()}
+            {this.renderConfirmDeleteModal()}
+        </>)
+    }
+
+    renderManageMonument() {
+        const { monument } = this.props;
+
+        return (
+            <div className="manage-monument">
+                <About monument={monument} contributions={monument.contributions} references={monument.references}
+                       header={monument.title} showHiddenFields hideExportToCSV hideTitle/>
+                {(monument.images && monument.images.length) ?
+                    <div className="images">
+                        {monument.images.map(image => (
+                            <div key={image.id} className="image" style={{backgroundImage: `url(${image.url})`}}/>
+                        ))}
+                    </div> : <></>
```

Comment:
Couldn't the false case of this ternary be `</>`? Or can you not do that with an empty tag?

Reply:
Yea it doesn't work with the empty tag. I could probably just return null though